packages/flutter/lib/src/animation/animations.dart

```diff
@@ -382,6 +396,11 @@ enum _TrainHoppingMode { minimize, maximize }
 /// object down.
 class TrainHoppingAnimation extends Animation<double>
   with AnimationEagerListenerMixin, AnimationLocalListenersMixin, AnimationLocalStatusListenersMixin {
+
+  /// Creates a train hopping animation.
```

Comment:
otherwise it means a train that hops animations, rather than an animation that hops trains.